common/men_at_arms_types/wc_regional_maa_types.txt

```diff
@@ -0,0 +1,189 @@
+﻿# standard costs
+@maa_buy_cost = 150
+@maa_low_maintenance_cost = 1.0
+@maa_high_maintenance_cost = 5.0
+
+arachnathid = { 
+	type = skirmishers
+	
+	damage = 13
+	toughness = 14
+	pursuit = 10
+	screen = 10
+	
+	terrain_bonus = {
+		hills = { damage = 3 toughness = 3 pursuit = 3 screen = 3 }
+		mountains = { damage = 4 toughness = 4 pursuit = 4 screen = 4 }
+		taiga = { damage = 3 toughness = 3 pursuit = 3 screen = 3 }
+	}
+	
+	winter_bonus = {
+		mild_winter = { damage = 2 toughness = 2 pursuit = 2 screen = 2 }
+		normal_winter = { damage = 3 toughness = 3 pursuit = 3 screen = 3 }
+		harsh_winter = { damage = 4 toughness = 4 pursuit = 4 screen = 4 }
+	}
+
+	counters = {
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_55_recruitment_cost }
+	low_maintenance_cost = { gold = cost_55_low_maint_cost }
+	high_maintenance_cost = { gold = cost_55_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = culture_ai_weight_arachnathid }
+	icon = arachnathid
+}
+dragonhawk_rider = {
+	type = light_aerial
+	
+	damage = 85
+	toughness = 70
+	pursuit = 66
+	screen = 35
+	
+	terrain_bonus = {
+		plains = { damage = 12 toughness = 5 }
```

Comment:
Some forest bonus since they live in Quel'thalas?

Reply:
Guess we left that out because fighting from an aerial unit in a forest is really hard. Massive aerials got a debuff there, light just aren't better or worse. You can't navigate between trees and hitting from above is not easy too, even if you're used to the terrain

Reply:
> You can't navigate between trees and hitting from above is not easy too, even if you're used to the terrain

Dragonhaws are used to navigate between trees and hitting from above and that's why they're used to the terrain.

Reply:
By what I see, dragonhawks don't have malus there compared to basic aerial unit. It's nice already, but to give better understanding I would give them at least small bonus, the same bonus as hills and plains. The player should see where they're supposed to be used.